src/main/java/org/commcare/formplayer/services/ScheduledTasks.java

```diff
@@ -41,6 +45,7 @@ public void purge() {
         log.info("Starting purge scheduled task.");
         doTimedPurge("formSession", formSessionService::purge);
         doTimedPurge("virtualDataInstance", virtualDataInstanceService::purge);
+        doMediaPurge("media", mediaHandler::purge);
```

Comment:
@shubham1g5   Originally I was using `doTimedPurge` to call `mediaHandler::purge` which required an `Instant` arg to be passed in. This gets rid of that required and unused arg. I'm curious if there are any different approaches that are preferable.

Reply:
Ahh I see what you meant earlier. I think in this case for the sake of consistency and avoiding code duplication, it's alright if we have unused `Instant` as an arg for `mediaHandler::purge`